documentation/proc-pages/development/standards.md

```diff
@@ -20,16 +26,46 @@ real(dp) :: b
 
 ```
 
+all new models should have their own function
+
+--------------------
+
 ## Naming conventions
 
+!!! quote "The Zen of Python"
+
+    *“Explicit is better than implicit.”*
+
+    *"Readability counts."*
+
+### Functions
+
+Use a lowercase word or words. Separate words by underscores(`_`) to improve readability.
+
+### Switches
+
+Switches should start with the `i_` prefix in their name, be of integer type and should be indexed from 0.
+Switches should by their nature not override other switches.
 
-### Case
+### Constants
 
-All variables should be lower case.
+Use an uppercase single letter, word, or words. Separate words with underscores to improve readability.
+
+Refrain from declaring or typing known numerical constants directly in the code. Instead call the value from `constants.f90`
+If the constants doesn't exist then add it with a source link and uncertainty value.
+
+### Variables
+
+Use a lowercase single letter, word, or words. Separate words with underscores to improve readability.
+If converting between units it may be required to have some capital letters at the end of the variable name to differentiate between different orders of magnitude, `m` and `M` etc.
+
+#### Variables representing fractions
+
+If a variable is intended to demonstrate a fraction of a value or distribution etc. Then it should start with the `f_` prefix.
```

Comment:
What about f-values, which also follow this pattern (at the moment)?

Reply:
Good point, I think for this case the majority of f-values have the `f` prefix without the underscore. There is currently quite a mish mash of fractional variables and f-values. Though hopefully soon with the number of f-values being reduced and variables being renamed to have the proper `f_` prefix, this should be more clear